zip-0226.rst

```diff
@@ -1,8 +1,305 @@
 ::
 
   ZIP: 226
-  Title: Zcash Shielded Assets - Transfers and Burns
-  Owners: Daniel Bennaroch
-  Status: Reserved
+  Title: Transfer and Burn of Zcash Shielded Assets
+  Owners: Daniel Benarroch <daniel@qed-it.com>
+          Pablo Kogan <pablo@qed-it.com>
+          Aurelien Nicolas <aurel@qed-it.com>
+  Credits: Daira Hopwood
+           Jack Grigg
+           Vivek Arte
+  Status: Draft
   Category: Consensus
+  Created: 2022-05-01
+  License: MIT
   Discussions-To: <https://github.com/zcash/zips/issues/618>
+
+
+Terminology
+===========
+
+The key word "MUST" in this document is to be interpreted as described in RFC 2119 [#RFC2119]_.
+
+The term "network upgrade" in this document is to be interpreted as described in ZIP 200 [#zip-0200]_.
+
+The term "Orchard" in this document is to be interpreted as described in ZIP 224 [#zip-0224]_.
+
+We define the following additional terms:
+
+- Asset: A type of note that can be transferred on the Zcash block chain, identified by the :math:`\mathsf{AssetId}` parameter.
+
+  - ZEC is the default (and currently the only defined) Asset for the Zcash mainnet.
+  - We use the term "custom Asset" to refer to any Asset other than ZEC.
+
+- Zcash Shielded Asset: an Asset with issuance defined on the Zcash block chain, and, for now, belonging to the Orchard shielded pool.
+- Wrapped Asset: a Zcash Shielded Asset with native issuance defined outside the Zcash block chain.
+- Split Input: an Action input identifying a Zcash Shielded Asset note, used to ensure that the output note of that Action is of a validly issued :math:`\mathsf{AssetId}` when there is no corresponding real input note to spend. The Action does not spend the note identified by the Split Input.
+- Split Action: a Zcash Shielded Asset Action that contains a Split Input.
+
+Abstract
+========
+
+ZIP 226 and ZIP 227 propose in conjunction the Zcash Shielded Assets (ZSA) protocol, which is an extension of the
+Orchard protocol that enables the creation, transfer, and burn of custom Assets on the Zcash chain. The creation of such Assets is defined
+in ZIP 227 [#zip-0227]_. The transfer and burn of such Assets is defined in ZIP 226 [#zip-0226]_.
```

Comment:
We use "Zcash Shielded Assets" to refer to the 
```suggestion
ZIP 226 and ZIP 227 propose in conjunction the Zcash Shielded Assets (ZSA) protocol &mdash; a set
of protocol features that enable the creation, transfer, and burn of Custom Assets on the Zcash chain.

Creation of such Assets is defined in ZIP 227 [#zip-0227]_. Transfer and burn of such Assets is defined
in ZIP 226 [#zip-0226]_. The ZSA protocol is proposed to be instantiated by a modification to the
Orchard protocol, as specified in these ZIPs.
```

Reply:
This is outdated, was adopted, and then has changed based on further reviews.